sunpy/timeseries/sources/eve.py

```diff
@@ -69,23 +85,12 @@ def peek(self, title='EVE/ESP Level1', **kwargs):
         """
 
         self._validate_data_for_plotting()
-
-        names = ('Flux \n 0.1-7nm', 'Flux \n 18nm', 'Flux \n 26nm', 'Flux \n 30nm', 'Flux \n 36nm')
-
         axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
```

Comment:
Same here?

Reply:
Similar to how GenericMap is written, I've set `axes` in peek and then passed it to plot. Do we not want this?

Reply:
Does plot not need an axes?

Reply:
If plot is called directly it still gets the axes from `plt.gca()` right

Reply:
Then do we need this line?

Reply:
Sorry, I misunderstood this part. There is redundant code here, the reason I did this was in all sources `axes` was set in `peek` and then passed to `plot`. We don't use `plt.gca()` here but for the sake of uniformity I left it as it is, if plot is called directly, the same code is repeated there as well. I'll make the change if this is redundant.

Reply:
I am not sure, it feels redundant. You might have to have a play around to decide.

Reply:
This is the only source that behaves this way, the other sources axes aren't defined in this manner so I guess I'll make an exception for this?

Reply:
I would like to know why we make this exception for this source.

Reply:
This source fetches the `axes` from a pandas DataFrame object which returns a list of 2 axes.

Reply:
Fair enough.